docs/assessment.md

```diff
@@ -350,6 +351,518 @@ CREATE TABLE [IF NOT EXISTS] table_name
+
+Workaround:
+Granting ANY FILE permissions will allow users to access untrusted databases. Note that ANY FILE will still enforce ACLs on any tables or external (storage) locations governed by Unity Catalog.
+This requires DBR 12.2 or later (DBR 12.1 or before is blocked on the network layer)
+
+[[back to top](#migration-assessment-report)]
+
+### AF305.1 - boto3
+
+The `boto3` library is used.
+
+Instance profiles (AWS) and service principals/managed identity (Azure) are not supported from the Python/Scala REPL or UDFs, e.g. using boto3 or s3fs, Instance profiles are only set from init scripts and (internally) from Spark. 
+
+**Workarounds**
+For accessing cloud storage (S3, ADLS, GCS), use storage credentials and external locations. 
+For accessing non-storage cloud services (e.g., AWS secrets manager, etc), use
+
+
+(AWS) Consider other ways to authenticate with boto3, e.g., by passing credentials from Databricks secrets directly to boto3 as a parameter, or loading them as environment variables. This page contains more information. Please note that unlike instance profiles, those methods do not provide short-lived credentials out of the box, and customers are responsible for rotating secrets according to their security needs.
+
+
+(Azure) Consider creating a Databricks secrets scope backed by Azure vault, and authenticate to external systems using secrets from that secrets scope
+
+
+(GCP) Some customers have used an external secrets store (like Hashicorp vault) and loaded credentials upon cluster startup into the cluster’s environment variables. Please note that those are usually not short-lived credentials, and customers are responsible for securing and rotation secrets according to their security needs.
+
+
+[[back to top](#migration-assessment-report)]
+
+### AF305.2 - boto3
+
+The `s3fs` library is used which provides posix type sematics for S3 access. s3fs is based on boto3 and has similar restrictions. The recommendation is to use EXTERNAL VOLUMES mapped to the fixed s3 storage location or Unity Catalog MANAGED VOLUMES.
+
+[[back to top](#migration-assessment-report)]
+
+### AF306.1 - dbutils...getContext
+
+The `dbutils.notebook.entry_point.getDbutils().notebook().getContext().toJson()` was found. This function may trigger a security exception in DBR 13.0 and above.
+
+[[back to top](#migration-assessment-report)]
+
+### AF306.2 - dbutils...getContext
+
+The `dbutils.notebook.entry_point.getDbutils().notebook().getContext()` was found. This function may trigger a security exception in DBR 13.0 and above.
+
+[[back to top](#migration-assessment-report)]
+
+### AF310.1 - credential passthrough
+
+The `dbutils.credentials.` is used for credential passthrough. This is not supported by Unity Catalog.
+
+[[back to top](#migration-assessment-report)]
+
+### AF311.1 - dbutils.fs
+
+The `dbutils.fs.` was found. DBUtils and other clients that directly read the data from cloud storage are not supported.
```

Comment:
dbutils.fs. still work with volumes though - so we should highlight that

Reply:
Will add.